Kudu.Core.Test/Jobs/ScheduleFacts.cs

```diff
@@ -31,12 +31,20 @@ public void ScheduleProvidesNextIntervalCorrectly(string cronExpression, string
 
             var triggeredJobSchedulerLoggerMock = new Mock<TriggeredJobSchedulerLogger>(String.Empty, mockEnvironment.Object, null);
             var schedule = Schedule.BuildSchedule(cronExpression, triggeredJobSchedulerLoggerMock.Object);
-            schedule.SetDateTimeProvider(new TestDateTimeNowProvider(DateTime.Parse("00:00:00 1/1/2015", CultureInfo.InvariantCulture, DateTimeStyles.AssumeLocal)));
+            var now = DateTime.Parse( "00:00:00 1/1/2015" , CultureInfo.InvariantCulture , DateTimeStyles.AssumeLocal );
```

Comment:
nit: looks like your editor settings are introducing a bunch of extra spaces in the code - please fix